ebu_tt_live/documents/ebutt3_deduplicator.py

```diff
@@ -0,0 +1,20 @@
+from datetime import timedelta
+import logging
```

Comment:
Non selective imports are typically bad. Try and only import what you need.
In this case `from logging import getLogger` would be preferred.